libraries/src/Installer/Installer.php

```diff
@@ -1709,12 +1709,15 @@ public function copyFiles($files, $overwrite = null)
 
 				if (!file_exists($filesource))
 				{
-					/*
-					 * The source file does not exist.  Nothing to copy so set an error
-					 * and return false.
-					 */
+					// The source file / folder does not exist. Nothing to copy so set an error.
 					Log::add(Text::sprintf('JLIB_INSTALLER_ERROR_NO_FILE', $filesource), Log::WARNING, 'jerror');
 
+					// Okay to ignore missing folder - it maybe an empty folder in the zip file.
```

Comment:
```suggestion
					// Okay to ignore missing folder - it may be an empty folder in the zip file.
```

it is a verb phrase in this usecase not an adverb

Reply:
Improved comment text.